auth/src/main/kotlin/gov/cdc/prime/reportstream/auth/config/SubmissionsConfig.kt

```diff
@@ -0,0 +1,13 @@
+package gov.cdc.prime.reportstream.auth.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.context.annotation.Configuration
+
+/**
+ * Configuration for Submissions microservice
+ */
+@Configuration
+@ConfigurationProperties(prefix = "submissions")
+data class SubmissionsConfig(
+    var baseUrl: String = "http://localhost:8080"
```

Comment:
Just confirming, the default isn't strictly required and this is ready from the config?

Reply:
There some weirdness without the default value. I can look more deeply into it. I think using those annotations is written to work well with standard java getters/setters and not as cleanly with a data class.

![Screenshot 2024-10-22 at 2 29 31 PM](https://github.com/user-attachments/assets/c60b797a-071a-4082-a0ed-480b8bb923a3)

Reply:
Looks like theres a workaround I'll try

https://stackoverflow.com/questions/76626376/configurationproperties-without-default-values-in-spring-boot-3-using-kotlin

Reply:
that workaround worked great. It also let me use vals instead of vars 👍